gr-analog/python/analog/wfm_rcv.py

```diff
@@ -17,7 +17,7 @@
 
 
 class wfm_rcv(gr.hier_block2):
-    def __init__(self, quad_rate, audio_decimation):
+    def __init__(self, quad_rate, audio_decimation, deemph_tau):
```

Comment:
Adding a default here will keep the API compatible.

Reply:
I wanted to avoid adding yet another place where a default is specified. It's already in both GRC yml and `fm_emph.py`.

You're saying it's worth it still?

Reply:
Since wfm_rcv could be used by external code (not just via GRC), I think so.